pkg/manifests/manifests.go

```diff
@@ -1731,6 +1737,27 @@ func (f *Factory) PrometheusUserWorkload(grpcTLS *v1.Secret) (*monv1.Prometheus,
 		p.Spec.Secrets = append(p.Spec.Secrets, getAdditionalAlertmanagerSecrets(alertManagerConfigs)...)
 	}
 
+	if trustedCABundleCM != nil {
```

Comment:
is there a real need to check for nil? if not, it might be more readable to inject the configmap along with the configuration of the startup probe (e.g. have only one loop iterating over the containers).

Reply:
ACK, Simon. I've pushed the snippet in the loop above.